src/sparql/mondo-excluded-subclasses.sparql

```diff
@@ -7,10 +7,6 @@ WHERE {
   ?childURI rdf:type owl:Class ;
     mondo:excluded_subClassOf ?parentURI .
 
-  # Extract the ID portion from the full URI and convert underscore to colon
-  BIND(STRAFTER(STR(?childURI), "http://purl.obolibrary.org/obo/") AS ?childWithUnderscore)
-  BIND(STRAFTER(STR(?parentURI), "http://purl.obolibrary.org/obo/") AS ?parentWithUnderscore)
-  # Replace underscores with colons
-  BIND(REPLACE(?childWithUnderscore, "_", ":") AS ?child)
-  BIND(REPLACE(?parentWithUnderscore, "_", ":") AS ?parent)
+  BIND(REPLACE(STR(?childURI), "http://purl.obolibrary.org/obo/MONDO_", "MONDO:") AS ?child)
```

Comment:
This is what I'd wanted to do as well, but for some reason (I don't remember why right now; perhaps I just had a misspelling though), I wasn't getting correct results. I just ran the outputs with your changes, though, and they are indeed correct and consistent with what I got in the commit before your change.